packages/calcite-components/src/components/block/block.scss

```diff
@@ -150,18 +161,18 @@ calcite-handle {
 }
 
 .toggle-icon {
-  @apply text-color-3
-  transition-color
+  @apply transition-color
   self-center
   justify-self-end
   duration-150
   ease-in-out;
 
   margin-inline-end: var(--calcite-spacing-md);
+  color: var(--calcite-block-header-text-color, var(--calcite-color-text-3));
```

Comment:
Unless a component has changed state at the :host level, a token used throughout the component must always have the same default. So lines 171, 175, and 211 won't work.